oauth2/handler.go

```diff
@@ -587,7 +587,7 @@ func (h *Handler) TokenHandler(w http.ResponseWriter, r *http.Request) {
 
 	accessResponse, err := h.r.OAuth2Provider().NewAccessResponse(ctx, accessRequest)
 	if err != nil {
-		x.LogError(r, err, h.r.Logger())
+		x.LogAudit(r, err, h.r.Logger())
```

Comment:
Why did you change this here? :)

Reply:
perhaps i'm mistaken but is this not where the error is logged in case credentials are invalid? my aim is for that error log to be downgraded to something less severe.

Reply:
That is correct, however there could be a number of issues which would warrant a error log such as a broken database connection and more. Maybe we should check for the error type and choose the logger only for cases where invalid credentials are sent?

Reply:
okay, my bad, i thought invalid credentials was the only remaining error here. will fix!